unity-renderer/Assets/UIComponents/Scripts/Components/ColorPicker/ColorSelector.cs

```diff
@@ -32,7 +32,10 @@ public void Cleanup()
         foreach (var colorToggle in colorToggles)
         {
 #if UNITY_EDITOR
-            DestroyImmediate(colorToggle.gameObject);
+            if (Application.isPlaying)
+                Destroy(colorToggle.gameObject);
+            else
+                DestroyImmediate(colorToggle.gameObject);
```

Comment:
```suggestion
            if !(Application.isPlaying)
            {            
                DestroyImmediate(colorToggle.gameObject);
                continue;
             }
```

Reply:
You are destroying already below, and since you are not continuing, it will break because you will try to destroy stuff twice.